scss/objects/_grid.scss

```diff
@@ -56,248 +56,248 @@ $DBbreakpoint-small: 480px !default;
+    // This might be overkill for some, but it's a time-saver for me.
+    .#{$DBgrid-column-namespace} {
+        box-sizing: border-box;
         display: inline-block;
-        margin-right: -.24em;
-      }
-
-      &.#{$DBgrid-column-namespace}--s-1-of-2 {
-        width: percentage(1/2);
-      }
-
-      &.#{$DBgrid-column-namespace}--s-1-of-3 {
-        width: percentage(1/3);
-      }
-
-      &.#{$DBgrid-column-namespace}--s-2-of-3 {
-        width: percentage(2/3);
-      }
-
-      &.#{$DBgrid-column-namespace}--s-1-of-4 {
-        width: percentage(1/4);
-      }
-
-      &.#{$DBgrid-column-namespace}--s-2-of-4 {
-        @extend .#{$DBgrid-column-namespace}--s-1-of-2;
-      }
-
-      &.#{$DBgrid-column-namespace}--s-3-of-4 {
-        width: percentage(3/4);
-      }
-    }
-  }
-  } // end if
-
-  // Centers the column in the grid and clears the row of all other columns
-  .#{$DBgrid-column-namespace}--centered {
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
-  }
-
-
-  // Displays the column as the first in its row
-  .#{$DBgrid-column-namespace}--d-first {
-    float: left;
-  }
-
-  // Displays the column as the last in its row
-  .#{$DBgrid-column-namespace}--d-last {
-    float: right;
-  }
-
-  // Removes gutters from the columns
-  &.grid--no-gutter {
-    margin-left: 0;
-    width: 100%;
+        margin-right: -.25em;
+        min-height: 1px;
+        padding-left: $DBgutter-width;
+        vertical-align: top;
+
+        [data-reactid] & {
+            // React spits out markup with no whitespace between elements, so the inline-block hack to remove spaces is no
+            // longer necessary.
+            margin-right: 0;
+        }
 
-    .#{$DBgrid-column-namespace} {
-      padding-left: 0;
-    }
-
-    .#{$DBgrid-column-namespace}--span-all {
-      margin-left: 0;
-      width: 100%;
-    }
-  }
-
-  // Align column to the bottom.
-  .#{$DBgrid-column-namespace}--ab {
-    vertical-align: bottom;
-  }
-
-  // Align column to the middle.
-  .#{$DBgrid-column-namespace}--am {
-    vertical-align: middle;
-  }
-
-}
+        @if ($DBresponsive-grid == true) {
+            @media (max-width: $DBbreakpoint-medium) {
+                display: block;
+                margin-left: 0;
+                margin-right: 0;
+                width: auto;
+            }
+
+            @media (max-width: $DBbreakpoint-medium) and (min-width: $DE-breakpoint-small) {
+                &[class*="#{$DBgrid-column-namespace}--m-"] {
+                display: inline-block;
+                margin-right: -.24em;
+                }
+
+                &.#{$DBgrid-column-namespace}--m-1-of-2 {
+                width: percentage(1/2);
+                }
+
+                &.#{$DBgrid-column-namespace}--m-1-of-3 {
+                width: percentage(1/3);
+                }
+
+                &.#{$DBgrid-column-namespace}--m-2-of-3 {
+                width: percentage(2/3);
```

Comment:
`2/3` should be written with a single space on each side of the operator: `2 / 3`